rpcd/playbooks/rpc-release.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Delete rpc-release file
+  hosts: hosts
+  gather_facts: false
+  tasks:
+    - name: Remove rpc-release file
```

Comment:
To satisfy the release test I suggest we simply link the two files. 

``` yaml
- name: Remove rpc-release file
   file:
     dest: /etc/rpc-release
     src: /opt/openstack-release
     state: link
     force: true
```

Reply:
Works for me, updated.